### ISSUE TITLE
About unexpected keys for hrnet

### ISSUE THREAD

**Issue by learning161** [open-mmlab/mmpose#356]
Hello, I'm new to mmpose.
When I train hrnetw32 with the default config. I get this warning message. I print the state_dict keys to debug, and fail to figure out why. Could you kindly lend me a hand here?
![image](https://user-images.githubusercontent.com/76098815/102330320-5b219700-3fc4-11eb-8915-7a2c05dd294a.png)

**Comment by jin-s13** [comment]
Hi，First that is not a bug. 
Since the image-net pretrained hrnet is trained for classification, there are some unexpected layers (e.g. classification head).

**Comment by learning161** [comment]
Yes, now I see. Cause I see the keys also include stage, it just confuses me.
By the way, I am trying to add integral loss for mmpose now. Just finish writing the code, test training now.
ref: https://arxiv.org/abs/1711.08229 https://github.com/JimmySuen/integral-human-pose

**Comment by innerlee** [comment]
Contributions are welcome :D
